sw/device/lib/runtime/BUILD

```diff
@@ -0,0 +1,98 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU")
+
+cc_library(
+    name = "epmp",
+    srcs = ["epmp.c"],
+    hdrs = ["epmp.h"],
+    target_compatible_with = [OPENTITAN_CPU],
+    deps = [
+        "//sw/device/lib/base/freestanding",
+    ],
+)
+
+cc_library(
+    name = "hart",
+    srcs = ["hart.c"],
+    hdrs = ["hart.h"],
+    target_compatible_with = [OPENTITAN_CPU],
+    deps = [
+        ":ibex",
+        "//sw/device/lib/arch:device",
+        "//sw/device/lib/base",
+        "//sw/device/lib/base/freestanding",
+    ],
+)
+
+cc_library(
+    name = "ibex",
+    srcs = ["ibex.c"],
+    hdrs = ["ibex.h"],
+    target_compatible_with = [OPENTITAN_CPU],
+    deps = [
+        "//sw/device/lib/arch:device",
+        "//sw/device/lib/base",
+    ],
+)
+
+cc_library(
+    name = "log",
+    srcs = ["log.c"],
+    hdrs = ["log.h"],
+    target_compatible_with = [OPENTITAN_CPU],
+    deps = [
+        "//sw/device/lib/arch:device",
+        "//sw/device/lib/base",
+        "//sw/device/lib/base/freestanding",
+        "//sw/device/lib/runtime:print",
+    ],
+)
+
+cc_library(
+    name = "otbn",
+    srcs = ["otbn.c"],
+    hdrs = ["otbn.h"],
+    deps = [
+        "log",
```

Comment:
`:log`, right...?

Reply:
yeah, I guess this isn't checked by bazel or buildifier, but :log is definitely correct